functorch/_src/aot_autograd.py

```diff
@@ -1442,6 +1442,27 @@ def aot_module_simplified(mod: nn.Module, *top_args, **top_kwargs) -> nn.Module:
     params_flat = tuple(params_flat)
     params_len = len(params_flat)
 
+
+    fake_mode = None
+    if "fake_mode" in top_kwargs and config.use_fake_tensor:
```

Comment:
At this point it's worth duplicating the necessary args/kwargs to this function.

Reply:
Ah yeah, another pr tho, I think?

Reply:
hmm nah ill do it now